libburn.yaml

```diff
@@ -0,0 +1,49 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/libburn/APKBUILD
+package:
+  name: libburn
+  version: 1.5.6
+  epoch: 0
+  description: Library for reading, mastering and writing optical discs
+  copyright:
+    - license: GPL-2.0-or-later
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - linux-headers
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 7295491b4be5eeac5e7a3fb2067e236e2955ffdc6bbd45f546466edee321644b
+      uri: https://files.libburnia-project.org/releases/libburn-${{package.version}}.tar.gz
+
+  - uses: autoconf/configure
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+subpackages:
+  - name: libburn-dev
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      runtime:
+        - libburn
+    description: libburn dev
+
+  - name: libburn-doc
+    pipeline:
+      - uses: split/manpages
+    description: libburn manpages
+
+update:
```

Comment:
Good news - looks like this is registered in release monitor:
- https://release-monitoring.org/project/1568/

So we could enable auto-updates here. Theres some other examples in wolfi we can refer to:
- https://github.com/search?q=repo%3Awolfi-dev%2Fos%20release-monitor%3A&type=code